src/components/views/settings/AvatarSetting.tsx

```diff
@@ -80,9 +86,9 @@ const AvatarSetting: React.FC<IProps> = ({ avatarUrl, avatarAltText, avatarName,
     return (
         <div className={avatarClasses}>
             {avatarElement}
-            <div className="mx_AvatarSetting_hover">
+            <div className="mx_AvatarSetting_hover" aria-hidden="true">
                 <div className="mx_AvatarSetting_hoverBg" />
-                <span>{_t("Upload")}</span>
+                <span id="hover-text">{_t("Upload")}</span>
```

Comment:
This component, if used twice in the page will not work as the document will have a duplicate identifier

- In an ideal world where we're using React 18, i'd suggest using [`useId()`](https://beta.reactjs.org/reference/react/useId), but we're not using that version yet
- For this scenario I believe we should namespace the `id` and maybe add a `Math.random()` suffix to the ID to avoid collision